app/components/NestedCheckboxes.tsx

```diff
@@ -13,6 +13,7 @@ type CheckboxArgs = Parameters<typeof Checkbox>
 type CheckboxProps = CheckboxArgs[0]
 interface NestedCheckboxProps extends CheckboxProps {
   nodes: CheckboxProps[]
+  counterhandler?: (arg: any) => void
```

Comment:
Is there a way to move this counter handler stuff to NoticeTypeCheckboxes?

Reply:
The function is defined on the NoticeTypeCheckboxes component level, and passed through the interface. I think this is necessary, since we are trying to have it act in response to a checkbox being clicked, a couple levels down component wise. I could try to make the interface's property more abstract, maybe something like 'childoncheckhandler' or something, if we think there is a need for that, but otherwise, I don't think the lines specifically in this file can be moved out.